gradle.properties

```diff
@@ -30,48 +30,19 @@ publicArtifactURL = https://download.corda.net/maven
 # dokka need more metaspace - https://github.com/Kotlin/dokka/issues/1405
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -XX:MaxMetaspaceSize=2g
 internalPluginVersion = 1.+
-dokkaVersion = 1.8.+
-detektPluginVersion = 1.22.+
-dependencyCheckVersion=0.46.+
 artifactoryPluginVersion = 4.28.2
-snakeyamlVersion=2.0
-
-# Logging
-slf4jVersion = 1.7.36
-
-# Main implementation dependencies
-avroGradlePluginVersion=1.3.0
-avroVersion = 1.11.3
-commonsCompressVersion = 1.24.0
-bouncycastleVersion = 1.73
-grgitPluginVersion = 5.2.0
-taskTreePluginVersion = 2.1.1
-javaxPersistenceApiVersion = 2.2
-jacksonVersion = 2.15.2
 
 # Testing
-assertjVersion = 3.24.+
 junitVersion = 5.10.0
```

Comment:
Some versions still left in here because the version catalog doesn't seem to work for the cordapp configuration plugin. This may need some looking into.